example/z-docker-opam/deploy.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+set -x
+
+docker-compose build
+docker-compose down
+docker-compose up --detach
```

Comment:
Should this also get `LOGGING_DRIVER=journald`?

Reply:
Done